cmake/FindCharm.cmake

```diff
@@ -68,7 +74,7 @@ include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(
   Charm
   FOUND_VAR CHARM_FOUND
-  REQUIRED_VARS CHARM_COMPILER CHARM_INCLUDE_DIRS CHARM_LIBRARIES
+  REQUIRED_VARS CHARM_COMPILER CHARM_INCLUDE_DIRS CHARM_LIBRARIES CHARM_EVERYLB
```

Comment:
The `REQUIRED_VARS` should be input parameters, i.e. the user may be asked to select them on the command line if they weren't found:

https://cmake.org/cmake/help/latest/module/FindPackageHandleStandardArgs.html?highlight=find_package_handle_standard_args#command:find_package_handle_standard_args

I think you should check the `CHARM_EVERYLB` variable yourself and error out with a useful message like "Make sure you have built the 'LIBS' target when building Charm++".